### DISCUSSION TITLE
Is there a way to skip already downloaded files?

### DISCUSSION THREAD

**Discussion by borderlands222** [Avnsx/fansly#76] [Q&A]
I would like to download a model's feed in multiple times but it downloads from the beginning every time. 
I don't know if update_recent_download is in that purpose but it looked really slow when I tried, like it was downloading the files again (I didn't go all the way through it).

Is there a way to just skip already downloaded files?

Thanks.

**Comment by Avnsx** [comment]
just use update_recent_download https://github.com/Avnsx/fansly/discussions/16#discussioncomment-2618679
